frontend/src/components/Profile/Profile.js

```diff
@@ -0,0 +1,88 @@
+import Axios from 'axios';
+import { Link } from "react-router-dom";
+import React, { useState, useEffect } from 'react'
+import '../../config';
```

Comment:
would it also be possible to move `../../config` to the top of the list and have a line separating the library imports from the stylesheet one please?